### ISSUE TITLE
show property does not work in b-alert

### ISSUE THREAD

**Issue by javiercbk** [bootstrap-vue/bootstrap-vue#106]
The following does not work
```html
<div>
<button @click="showAlert">showAlert</button>
<b-alert :show="condition" state="danger" dismissible>
  message
</b-alert>
</div>
```

```js
export default {
  data() {
    return {
      condition: false,
    };
  },
  methods: {
    showAlert() {
      this.condition = true;
    }
  }
}
```

The `alert.vue` file watches the show expression but never updates the `localShow`. It only defines the initial value, but never updates it. This might be the issue

```js
data() {
  return {
    localShow: this.show
  }
},
watch: {
  show: function (newValue, oldValue) {
    if (this.dismissAfterSeconds && newValue == true && oldValue == false) {
      this.dismissCounter();
    }
  }
},
```

**Comment by pi0** [comment]
will be fixed soon

**Comment by pi0** [comment]
@javiercbk Just fixed in 0.8.0 :)  (Please also check out docs source code and note the trick for correct usage.) [Docs](https://bootstrap-vue.github.io/docs/components/alerts)
